targets/targets.mk

```diff
@@ -9,13 +9,16 @@ $(eval $(call GluonTarget,x86,generic))
 $(eval $(call GluonTarget,x86,geode))
 $(eval $(call GluonTarget,x86,64))
 
+ifeq ($(GLUON_WLAN_MESH),11s)
```

Comment:
I would expect that these targets are also build in case `BROKEN` is set (concerning `GLUON_WLAN_MESH = ibss`).

Reply:
i second that, there should be an OR (pseudocode: if GLUON_WLAN_MESH = 11s OR BROKEN = 1 )